hanami-utils.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.3.0'
 
   spec.add_dependency 'transproc', '~> 1.0'
+  spec.add_dependency 'wisper', '~> 2.0.0'
```

Comment:
Thanks. I like this idea. So this way we can get rid from hard-coded dependency and let clients decide which backend they want to use.